docs/docs/50-user-guide/60-reference-docs/30-promotion-steps/git-clone.md

```diff
@@ -24,6 +24,10 @@ multiple working trees.
 |------|------|----------|-------------|
 | `repoURL` | `string` | Y | The URL of a remote Git repository to clone. |
 | `insecureSkipTLSVerify` | `boolean` | N | Whether to bypass TLS certificate verification when cloning (and for all subsequent operations involving this clone). Setting this to `true` is highly discouraged in production. |
+| `user` | `[]object` | N | User information for the Git operations. If not specified, system-level Git user details will be used. |
+| `user.name` | `string` | Y | The name of the user performing the Git operations. |
+| `user.email` | `string` | Y | The email of the user performing the Git operations. |
+| `user.signingKey` | `string` | N | The GPG signing key for the user. This field is optional. |
```

Comment:
May need some tweaks depending on how https://github.com/akuity/kargo/pull/3922/files#r2060546871 is resolved.

Don't forget to document these fields for `git-commit` also to account for https://github.com/akuity/kargo/issues/3813#issuecomment-2828788694.